### ISSUE TITLE
Same server can be added multiple times

### ISSUE THREAD

**Issue by robbi5** [blindsidenetworks/scalelite#152]
While trying to automate our bbb- and scalelite setup, I've found out that the servers model doesn't seem to use an unique constraint for URL and Secret, so that the same server got added multiple times:

```
$ ./bin/rake servers:add[https://bbb01.example.org/bigbluebutton/api,SECRETSECRETSECRET]
OK
id: ddf9c234-5bae-4b32-9ebf-11848d13d6b2
$ ./bin/rake servers:add[https://bbb01.example.org/bigbluebutton/api,SECRETSECRETSECRET]
OK
id: ba21e2f4-1fa1-4706-a09c-a71d7d23a129
$ ./bin/rake servers
id: ddf9c234-5bae-4b32-9ebf-11848d13d6b2
	url: https://bbb01.example.org/bigbluebutton/api
	secret: SECRETSECRETSECRET
	disabled
	load: unavailable
	offline
id: ba21e2f4-1fa1-4706-a09c-a71d7d23a129
	url: https://bbb01.example.org/bigbluebutton/api
	secret: SECRETSECRETSECRET
	disabled
	load: unavailable
	offline
```

I'm not sure if this is known and on purpose - if so, just close this issue :) But if not, maybe introducing an unique constraint for URL + Secret looks like a good idea to me.

**Comment by farhatahmad** [comment]
Hi @robbi5 

Thanks for reporting this. This probably shouldn't be happening so I'll tag it as a bug

**Comment by kepstin** [comment]
As part of the original design, I didn't think this was important... An administrator should just not add duplicate servers if they don't want them. (I can't think of any reasonable case to have them, but it doesn't break anything in scalelite if you do have them).

That said, we have a potential solution to this problem in #205 - by making the hostname itself the id rather than generating a unique uuid, it's much easier to ensure uniqueness.

**Comment by jfederico** [comment]
This was solved in v1.0.12 and it is part of v1.1
